source/camera/basic/capture.py

```diff
@@ -1,19 +1,30 @@
-"""Capture sample."""
+"""
+This example shows how to capture point clouds, with color, from the Zivid camera.
+"""
+
 import datetime
 import zivid
 
 
 def _main():
     app = zivid.Application()
+
+    print("Connecting to camera")
     camera = app.connect_camera()
 
+    print("Configuring settings")
     settings = zivid.Settings()
     settings.acquisitions.append(zivid.Settings.Acquisition())
     settings.acquisitions[0].aperture = 5.66
-    settings.acquisitions[0].exposure_time = datetime.timedelta(microseconds=8333)
+    settings.acquisitions[0].exposure_time = datetime.timedelta(microseconds=6500)
+    settings.processing.filters.outlier.removal.enabled = True
+    settings.processing.filters.outlier.removal.threshold = 5.0
```

Comment:
Was this on purpose? Extending how many settings you set.

Reply:
Yes, to make it equivalent to C++ and C#